buildSrc/src/main/kotlin/com/datadog/gradle/plugin/jsonschema/generator/ClassDeserializerGenerator.kt

```diff
@@ -167,7 +167,7 @@ class ClassDeserializerGenerator(
     ) {
         val opt = if (nullable) "?" else ""
         beginControlFlow(
-            "$assignee = $getter$opt.asJsonArray$opt.let { %L ->",
+            "$assignee = $getter$opt.asJsonArray$opt.let·{·%L·->",
```

Comment:
why there is a change with middle dot instead of space here and in `ClassGenerator`?

Reply:
This is part of KotlinPoet internals to mark non-breaking space. This ensures that the KotlinPoet file writer won't break a line on any of those space. 
This is because some classNames get very large and the line breaking happens at an arbitrary size without knowledge of the syntax, leading to code that don't compile